Documentation/Blazorise.Docs/Pages/Docs/Components/Skeletons/SkeletonPage.razor

```diff
@@ -0,0 +1,88 @@
+﻿@page "/docs/components/skeleton"
+
+<Seo Canonical="/docs/components/skeleton" Title="Blazorise Skeleton system" Description="Learn to use and work with the Blazorise Skeleton component, loading placeholders for your components or pages to indicate something may still be loading." />
+
+<DocsPageTitle Path="Components/Skeleton">
+    Blazorise Skeleton component
+</DocsPageTitle>
+
+<DocsPageLead>
+    Use loading placeholders for your components or pages to indicate something may still be loading.
```

Comment:
And just serving as a placeholder is that a viable case?

Reply:
That's why they are used, yes.